plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1392,6 +1392,32 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 			},
 		},
 	}
+	deletedPVCPod := &api.Pod{
```

Comment:
The pod `deletedPVCPod` refers to a PVC that is missing and exercises the condition below.
`c.pvcInfo.GetPersistentVolumeClaimInfo(namespace, pvcName) != nil`

The pod `deletedPVPod` refers to a PVC that exists but references a PV that is missing and hence exercises the condition below.
`c.pvInfo.GetPersistentVolumeInfo(pvName) != nil`

In both cases, I have verified the code path execution by manually modifying the predicate to return error vs nil for both these conditions. I have added test cases/scenarios to verify that having existing pods with missing PV or missing PVC result in these PVs being counted towards the node limit.